spec/index.html

```diff
@@ -729,9 +731,10 @@ <h2>Literals</h2>
 
     <p>A literal is a <dfn>language-tagged string</dfn> if the third element
       is present and the fourth element is not present.
-      Lexical representations of language tags MAY be converted
-      to lower case.
-      The value of language tags is always treated as being in lower case.</p>
+      Lexical representations of language tags
+      MUST be case normalized
+      and MAY (preferably) be converted to lower case.
```

Comment:
This basically reverts the wording that that used previously.

```suggestion
       Lexical representations of language tags
       MAY be converted to lower case.
```

Reply:
Alternatively:

```suggestion
       Lexical representations of language tags
       MAY be case normalized,
       (for example, by converting to lower case).
```